core/lib/spree/testing_support/dummy_app.rb

```diff
@@ -51,7 +51,13 @@ class Application < ::Rails::Application
     config.cache_classes = true
     config.action_controller.allow_forgery_protection = false
     config.action_controller.default_protect_from_forgery = false
-    config.action_controller.perform_caching = true
+
+    # In the test environment, we use the `caching: true` RSpec metadata to
```

Comment:
I'd put this block after the rest of the lines that are within the ` # Make the test environment more production-like:` block

Reply:
Okay, I'll make the change.